### ISSUE TITLE
Error to run BiocManager::install(). 

### ISSUE THREAD

**Issue by Sanchezillana** [Bioconductor/BiocManager#30]
I have R version 3.5.1. After installing BiocManager, I tried to install bioconductor and I obtin the following error: 

> BiocManager::install()
Error: invalid version specification ‘Bioconductor version cannot be validated; no internet connection?.

**Comment by LiNk-NY** [comment]
Hi Illana, @Sanchezillana 
Please check your options and make sure you don't have 
`BIOCONDUCTOR_ONLINE_VERSION_DIAGNOSIS` set to `FALSE` in your options or
in your `~/.Rprofile`. 
Also make sure you have an active internet connection when running the command. 

Best regards,
Marcel

**Comment by Sanchezillana** [comment]
> 
> 
> Hi Illana, @Sanchezillana
> Please check your options and make sure you don't have
> `BIOCONDUCTOR_ONLINE_VERSION_DIAGNOSIS` set to `FALSE` in your options or
> in your `~/.Rprofile`.
> Also make sure you have an active internet connection when running the command.
> 
> Best regards,
> Marcel

I do not have this parameter. I have just cheked it and I obtain the same error...

**Comment by LiNk-NY** [comment]
Please also make sure that you are not behind a firewall.

**Comment by Sanchezillana** [comment]
I'm not behind a firewall. I can install the BiocManager from CRAN with install.packages...

**Comment by mtmorgan** [comment]
Does
```
    config <- "https://bioconductor.org/config.yaml"
    readLines(config)
```
work? Can you provide the output of `traceback()` after the error occurs?

**Comment by Sanchezillana** [comment]
> traceback()

The requested outputs are: 

> BiocManager::install()
Error: invalid version specification ‘Bioconductor version cannot be validated; no internet connection?’
> traceback()
6: stop(gettextf("invalid version specification %s", paste(sQuote(unique(x[!ok])), 
       collapse = ", ")), call. = FALSE, domain = NA)
5: .make_numeric_version(x, strict, .standard_regexps()$valid_package_version, 
       "package_version")
4: package_version(x)
3: .package_version(version)
2: .version_validate(version)
1: BiocManager::install()

And, 

R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=Spanish_Spain.1252  LC_CTYPE=Spanish_Spain.1252    LC_MONETARY=Spanish_Spain.1252
[4] LC_NUMERIC=C                   LC_TIME=Spanish_Spain.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] BiocManager_1.30.3 compiler_3.5.1     tools_3.5.1       

Yes, I am using RStudio. The other outputs: 
> .libPaths()
[1] "C:/Users/73596984c/Documents/R/win-library/3.5"
[2] "C:/Program Files/R/R-3.5.1/library"     

I don't have the BiocVersion, but I have BiocManager: 

> installed.packages()["BiocManager",]
                                                  Package 
                                            "BiocManager" 
                                                  LibPath 
                     "C:/Program Files/R/R-3.5.1/library" 
                                                  Version 
                                                 "1.30.3" 
                                                 Priority 
                                                       NA 
                                                  Depends 
                                           "R (>= 3.5.0)" 
                                                  Imports 
                                                  "utils" 
                                                LinkingTo 
                                                       NA 
                                                 Suggests 
"BiocStyle, BiocVersion, remotes, testthat, knitr, withr" 
                                                 Enhances 
                                                       NA 
                                                  License 
                                           "Artistic-2.0" 
                                          License_is_FOSS 
                                                       NA 
                                    License_restricts_use 
                                                       NA 
                                                  OS_type 
                                                       NA 
                                                   MD5sum 
                                                       NA 
                                         NeedsCompilation 
                                                     "no" 
                                                    Built 
                                                  "3.5.1" 

Thank you very much.

**Comment by mtmorgan** [comment]
Does
```
    config <- "https://bioconductor.org/config.yaml"
    readLines(config)
```
work?

**Comment by mtmorgan** [comment]
Please see the possible fix in issue #32.

**Comment by vanhoan310** [comment]
I got the same problem because I use R on conda environment. I fixed it add file "cacert.pem" (in the anaconda environment) to R as follow.
Sys.setenv(CURL_CA_BUNDLE = "//path//to//cacert.pem file")
For details please see in here: http://tinyheero.github.io/2016/11/12/bioconductor-conda.html
